tests/pytests/test_wildcard.py

```diff
@@ -4,168 +4,152 @@
 import time
 
 def testSanity(env):
```

Comment:
In general I would prefer to split it to small test, if this test fails its hard to debug and understand why ..